ipaplatform/base/tasks.py

```diff
@@ -211,6 +211,10 @@ def remove_httpd_service_ipa_conf(self):
         """Remove configuration of httpd service of IPA"""
         raise NotImplementedError()
 
+    def configure_httpd_wsgi_conf(self):
+        """Configure WSGI for correct Python version"""
+        # only used on Fedora, NOP on other platforms
+
```

Comment:
`raise NotImplementedError()` missing

Reply:
Nope, I did this on purpose. The function is not implemented on any platform except Fedora.